src/param.F

```diff
@@ -273,7 +275,7 @@ subroutine param(namelist_input,                                  &
       call MPI_BCAST(idiss    ,1,MPI_INTEGER,0,MPI_COMM_WORLD,ierr)
       call MPI_BCAST(efall    ,1,MPI_INTEGER,0,MPI_COMM_WORLD,ierr)
       call MPI_BCAST(rterm    ,1,MPI_INTEGER,0,MPI_COMM_WORLD,ierr)
-      call MPI_BCAST(wbc      ,1,MPI_INTEGER,0,MPI_COMM_WORLD,ierr)
+      !KLUDGE call MPI_BCAST(wbc      ,1,MPI_INTEGER,0,MPI_COMM_WORLD,ierr)
```

Comment:
why is this line commented out?

Reply:
I comment out this particular call for the wbc variables because I added one at the end of the #ifdef MPI block.  So this was the test variable.

Reply:
I see. Thanks!